Mathlib/Analysis/SpecialFunctions/Complex/Log.lean

```diff
@@ -278,8 +278,8 @@ lemma Real.rexp_tsum_eq_tprod (f : ι → α → ℝ) (hfn : ∀ x n, 0 < f n x)
   congr
   exact funext fun x ↦ exp_log (hfn a x)
 
-lemma Real.summable_cexp_multipliable (f : ι → α → ℝ) (hfn : ∀ x n, 0 < f n x)
-    (hf : ∀ x : α, Summable fun n => log (f n x)) (a : α) : Multipliable fun b ↦ f b a := by
+lemma Real.summable_multipliable (f : ι → α → ℝ) (hfn : ∀ x n, 0 < f n x)
```

Comment:
The statement here looks strange to me - why is the `α` argument relevant? It looks like we're essentially working with a sequence of functions indexed by `ι` and `α`, but only ever summing / multiplying over the `ι` family.